bay/docker/runner.py

```diff
@@ -310,8 +310,12 @@ def handler():
                     elif status is True:
                         break
                     elif status is False:
-                        raise DockerRuntimeError(
+                        message = "{}\n\n{}".format(
                             "Container {} failed to boot!".format(instance.container.name),
+                            self.host.client.logs(instance.name['Names'][0], tail=10),
```

Comment:
Does `instance.name` not work here? That's meant to be the string name of the container at this point, and it's being used in the line right above it.

Reply:
the container we were testing on was core-frontend. 'instance.name' comes out as would be expected 'core-frontend' but host.client.logs spit out an error stating cannot find container with name core-frontend. We found that instead 'eventbrite.core-frontend.1' worked while 'core-frontend' did not.

Reply:
Weird, because `instance.name` should be `eventbrite.core-frontend.` while `instance.container.name` should be `core-frontend`...